### PR TITLE
Update perms for Nightcrawler

### PR DESCRIPTION
**Pull request by ThisChicken**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - worldedit.navigation.jumpto.command
+        - ch.alias.invis
+        - ch.alias.vis
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: evidence required

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+        -essentials.protect.damage.fall
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: evidence required for no fall damage

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: https://www.youtube.com/watch?v=SXksSlMD5nk

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: cannot, ill take it out

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: approved

**Pull request merged**